### ISSUE TITLE
No option 'cc' in section: 'compiler'

### ISSUE THREAD

**Issue by stehol01** [serge-sans-paille/pythran#655]
Hy pythran users,

I would like to try pythran but i encounter a problem if i do so. If i type pythran --help in my WinPython Command prompt. The following error occurs:

WARNING: Disabling color, you really want to install colorlog.

Traceback (most recent call last):
  File "<pyshell#0>", line 1, in <module>
    import pythran
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\__init__.py", line 39, in <module>
    from pythran.toolchain import (generate_cxx, compile_cxxfile, compile_cxxcode,
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\toolchain.py", line 6, in <module>
    from pythran.backend import Cxx
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\backend.py", line 8, in <module>
    from pythran.analyses import ArgumentEffects, BoundExpressions, Dependencies
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\analyses\__init__.py", line 12, in <module>
    from .aliases import Aliases, StrictAliases
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\analyses\aliases.py", line 4, in <module>
    from pythran.intrinsic import Intrinsic, Class, UnboundValue
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\intrinsic.py", line 5, in <module>
    from pythran.types.signature import extract_combiner
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\types\signature.py", line 4, in <module>
    from pythran import cxxtypes
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\cxxtypes.py", line 5, in <module>
    from pythran.config import cfg
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\config.py", line 107, in <module>
    '.pythranrc')
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\config.py", line 30, in init_cfg
    value = cfgp.get('compiler', key)
  File "C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\ConfigParser.py", line 618, in get
    raise NoOptionError(option, section)
NoOptionError: No option 'cc' in section: 'compiler'

I use python 2.7. 

Thank you for your help. :)

**Comment by serge-sans-paille** [comment]
Hi @stehol01 . We have very few feedbacks on Windows, so get ready for some trouble!

That being said (did I mention _welcome_ ?) It means that somewhere, there is a `.pythranrc` or a `pythran-win32.cfg` that lacks the `CC=` entry. Even if the entry is empty, it should be okay.

The default file is indeed:

```[compiler]
defines=
undefs=
include_dirs=
libs=
library_dirs=
cflags=-std=c++11
ldflags=
blas=
CC=
CXX=
```

What is the content of `C:\WinPython\WinPython-64bit-2.7.13.1Zero\python-2.7.13.amd64\lib\site-packages\pythran\pythran-win32.cfg` ?

**Comment by serge-sans-paille** [comment]
Should be fixed by #739
